### DISCUSSION TITLE
Android/iOS route audio through Earpiece

### DISCUSSION THREAD

**Discussion by Baklap4** [CommunityToolkit/Maui#1401] [Q&A]
I'm creating an app where the audio will have to be played through the earpiece but i'm not sure how this is done as i don't see any methods exposing the `STREAM_TYPE_VOICE_CALL` for example for ExoPlayer. Not quite sure what the equivalent is for iOS as i haven't started researching this.

I explicitly don't want to route the audio through the normal output since the app should not disturb other visitors of the location where the app is used.

Is this possible? Or is there a way to get to the Exoplayer within a maui xaml or code behind page?

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
